app/src/pages/playground/PlaygroundDatasetExamplesTable.tsx

```diff
@@ -78,7 +78,7 @@ import {
 } from "./PlaygroundToolCall";
 import { getChatCompletionOverDatasetInput } from "./playgroundUtils";
 
-const PAGE_SIZE = 100;
+const PAGE_SIZE = 10;
```

Comment:
Should this match the 20 defined in the query below?

Reply:
it doesn't need to, fetch 20 initially then 10 for each page. Doesn't make a big difference either way